docs/source/conf.py

```diff
@@ -289,5 +296,8 @@
 #texinfo_no_detailmenu = False
 
 intersphinx_mapping = {
-    'ipython': ('http://ipython.org/ipython-doc/dev/', None)
+    'ipython': ('http://ipython.org/ipython-doc/dev/', None),
+    'ipykernel': ('http://ipython.org/ipykernel-doc/dev/', None),
+    'nbconvert': ('http://jupyter.org/nbconvert-doc/dev/', None),
+    'nbformat': ('http://jupyter.org/nbformat-doc/dev/', None)
```

Comment:
Leave it for now, as it's pointing to something that works. We can always change it later.